modules/seqexec/server/src/main/scala/seqexec/server/System.scala

```diff
@@ -12,11 +12,11 @@ trait System[F[_]] {
   /**
     * Called to configure a system
     */
-  def configure(config: Config): SeqActionF[F, ConfigResult[F]]
+  def configure(config: Config): F[ConfigResult[F]]
```

Comment:
This is a key change. It also means the caller is responsible for error handling